### ISSUE TITLE
Question the team's diversity

### ISSUE THREAD

**Issue by Schepp** [viraptor/reverse-interview#43]
I would be interested in a team's balance in terms of female/male, old/young, PoC, LGTBQ, abled/disabled and religion – as I wouldn't wanna work in a white straight young abled christian males only environment.

**Comment by Addvilz** [comment]
Age, sexual orientation, race, disability status, and religious beliefs are all part of protected private information of an individual. As an example, nowhere in the EU is it allowed for the employer to possess or even ask such information from their employees, not to mention hand it out to some third party, like yourself. The only exception for what you listed is disability status, and even then only in certain scenarios. The same applies to any other medical or situational condition that is not directly related to the job position and requirements for performance. Regarding age specifically, employers have the right to know the date of birth for some specific reasons, but they are absolutely not allowed to go around and advertise the age of their employees just like that. Some of these could be disclosed with the express permission of each individual employee, but that is unlikely to happen.

Bottom line, you can ask - but no employer would be able to really answer as doing so would almost certainly violate the privacy of their existing employees.

And no, you can't even have "statistics" because it could be reasonable to assume you could identify the individuals behind the statistics, given the limited sample size behind those statistics, regardless if you would or would not end up working with that employer.

For these reasons, including such questions would be pointless.

**Comment by viraptor** [comment]
Do you think this satisfies your question: (it's in the social issue category at the moment)

> What's the status of / view on diverse hiring?

**Comment by Schepp** [comment]
Sure! I think it is always open to debate into how much detail you may want to go here.

Thank you <3
